### ISSUE TITLE
"Internal server error" when using flok-web from npm package

### ISSUE THREAD

**Issue by santoluigi** [munshkr/flok#86]
hi all

when i try to use flok in a local server i get "Internal Server Error"...

this is the terminal output:

> Listening on http://0.0.0.0:3000
TypeError: Cannot read property 'filter' of undefined
    at NextScript.getPolyfillScripts (/usr/local/lib/node_modules/flok-web/.next/server/static/t5tFuLg6rvyJANM_y3DHp/pages/_document.js:651:26)
    at NextScript.render (/usr/local/lib/node_modules/flok-web/.next/server/static/t5tFuLg6rvyJANM_y3DHp/pages/_document.js:750:41)
    at processChild (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3134:18)
    at resolve (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:2960:5)
    at ReactDOMServerRenderer.render (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3435:22)
    at ReactDOMServerRenderer.read (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:29)
    at renderToStaticMarkup (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:4004:27)
    at renderDocument (/usr/local/lib/node_modules/flok-web/node_modules/next/dist/next-server/server/render.js:3:640)
    at renderToHTML (/usr/local/lib/node_modules/flok-web/node_modules/next/dist/next-server/server/render.js:52:103)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
TypeError: Cannot read property 'filter' of undefined
    at NextScript.getPolyfillScripts (/usr/local/lib/node_modules/flok-web/.next/server/static/t5tFuLg6rvyJANM_y3DHp/pages/_document.js:651:26)
    at NextScript.render (/usr/local/lib/node_modules/flok-web/.next/server/static/t5tFuLg6rvyJANM_y3DHp/pages/_document.js:750:41)
    at processChild (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3134:18)
    at resolve (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:2960:5)
    at ReactDOMServerRenderer.render (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3435:22)
    at ReactDOMServerRenderer.read (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:29)
    at renderToStaticMarkup (/usr/local/lib/node_modules/flok-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:4004:27)
    at renderDocument (/usr/local/lib/node_modules/flok-web/node_modules/next/dist/next-server/server/render.js:3:640)
    at renderToHTML (/usr/local/lib/node_modules/flok-web/node_modules/next/dist/next-server/server/render.js:52:103)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

**Comment by munshkr** [comment]
Hi @santoluigi , thanks for reporting this, it's an issue with the flok-web package. In the mean time I suggest you start the webserver from the repository instead of using the package. You should follow these steps:

```
git clone https://github.com/munshkr/flok.git
cd flok/packages/web
yarn
yarn dev
```

You may need to install [Yarn](https://classic.yarnpkg.com/en/docs/install/#debian-stable). Let me know if that works for you!

**Comment by kornakar** [comment]
Had the same error from running the flok-web package.

Cloned the repository and run yarn (and yarn dev) in the flok/packages/web directory, but it gives a compilation error.

Then I tried running 'yarn' in the flok directory itself (so not in packages\web), and then the build succeeded.

**Comment by munshkr** [comment]
Fixed on the latest version (0.3.16), I had to pin the Nextjs version to a specific one.
